test/Tests/src/Semantic/Equals_Spec.enso

```diff
@@ -3,8 +3,10 @@ from Standard.Base import all
 from Standard.Test import Test, Test_Suite
 import Standard.Test.Extensions
 
+polyglot java import java.math.BigInteger as JavaBig
```

Comment:
I'd have called it `JavaBigInteger` but that's only because truncated names don't appear to be our style.

Reply:
`Java_Big_Integer` be it then since 2d8df09. Thank you.